tests/unit_tests/pretrain/test_pretrain_qwen3_next.py

```diff
@@ -0,0 +1,159 @@
+import os
+import pytest
+import subprocess
+
+
+def get_bash_cmd(model_args, env_args = None, num_gpus_per_node=8):
```

Comment:
FYI< this code snippit might be useful to you: https://github.com/facebookresearch/habitat-sim/blob/6c26c4f9ae10ad7f534aa9e59a84dff00782741e/tests/helpers/utils.py#L1